Gemfile

```diff
@@ -2,6 +2,8 @@ source "https://rubygems.org"
 
 gemspec
 
+gem 'hammer_cli_foreman', github: 'theforeman/hammer-cli-foreman', branch: 'master'
```

Comment:
I wonder why do we need this here, when we already have this dependency in .gemspec file?